app/_locales/en/messages.json

```diff
@@ -1909,6 +1915,12 @@
   "pending": {
     "message": "Pending"
   },
+  "pendingTransaction": {
+    "message": "You have ($1) pending transaction(s)."
```

Comment:
Generally with keys that contain variables we add a `description` key to the object explaining what the variable refers to.

Reply:
I have more PRs lined up and I will address it there